### PR TITLE
[refactor] Separate out queries

### PR DESCRIPTION
**Pull request by MikeVedsted**

### CODE REVIEW

File: src/services/request.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    const participant = createParticipant.rows[0]
+    return { message: 'Successfully accepted', participant }
+  } catch (error) {
+    return error
+  }
+}
+
+export default {
+  acceptRequest
+}
```
Comment (Rostislav013) [inline src/services/request.ts:17]: Does it needed to be returned as object?

Comment (MikeVedsted) [reply src/services/request.ts:17]: What do you mean? This is how we do it for all services. There will be more functions to export here later

Comment (Rostislav013) [reply src/services/request.ts:17]: I thought we can return like `export default acceptRequest`

**Review by Rostislav013 — APPROVED** [review]

File: src/services/request.ts (hunk 1 above)
Comment (MikeVedsted) [reply src/services/request.ts:17]: Sure, as long as there is one but there are already more in the other PRs, so no need to optimize prematurely here 😉

**Review by Chiranjibichapagain — APPROVED** [review]
:+1:

**Pull request merged**